### PR TITLE
Update sensitive-information-type-learn-about.md to include a note for customers moving from existing number based system to upgraded confidence levels

### PR DESCRIPTION
**Pull request by arastogi205**
@chrfox - we have had a couple of enquiries from existing customers on how their existing policies & custom SITs will get impacted and if they need to take an action to move to the upgraded confidence levels. I have added this not in order to address the same. Please review and suggest changes.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/sensitive-information-type-learn-about.md [hunk 1]
```diff
@@ -119,6 +119,11 @@ A high confidence level returns the fewest false positives but might result in m
 
 You should use high confidence level patterns with low counts, say five to ten, and low confidence patterns with higher counts, say 20 or more.
 
+> [!NOTE]
+> If you have existing policies or custom SITs defined using number based confidence levels(also know as accuracy), then they will automatically be mapped to the three discrete confidence levels - low confidence, medium confidence & high confidence across the SCC UI.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-learn-about.md]:
```suggestion
> If you have existing policies or custom sensitive information types (SITs) defined using number-based confidence levels (also know as accuracy), they will automatically be mapped to the three discrete confidence levels; low confidence, medium confidence, and high confidence, across the Security @ Compliance Center UI.
```

File: microsoft-365/compliance/sensitive-information-type-learn-about.md [hunk 2]
```diff
@@ -119,6 +119,11 @@ A high confidence level returns the fewest false positives but might result in m
...
+> - All policies with min accuracy or custom SIT patterns with confidence levels between 76 & 100 will be mapped to high confidence.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-learn-about.md]:
```suggestion
> - All policies with minimum accuracy or custom SIT patterns with confidence levels of between 76 and 100 will be mapped to high confidence. 
```

File: microsoft-365/compliance/sensitive-information-type-learn-about.md [hunk 3]
```diff
@@ -119,6 +119,11 @@ A high confidence level returns the fewest false positives but might result in m
 
 You should use high confidence level patterns with low counts, say five to ten, and low confidence patterns with higher counts, say 20 or more.
 
+> [!NOTE]
+> If you have existing policies or custom SITs defined using number based confidence levels(also know as accuracy), then they will automatically be mapped to the three discrete confidence levels - low confidence, medium confidence & high confidence across the SCC UI.
+> - All policies with min accuracy or custom SIT patterns with confidence levels between 76 & 100 will be mapped to high confidence. 
+> - All policies with min accuracy or custom SIT patterns with confidence levels between 66 & 75 will be mapped to medium confidence.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-learn-about.md]:
```suggestion
> - All policies with minimum accuracy or custom SIT patterns with confidence levels of between 66 and 75 will be mapped to medium confidence.
```

File: microsoft-365/compliance/sensitive-information-type-learn-about.md [hunk 4]
```diff
@@ -119,6 +119,11 @@ A high confidence level returns the fewest false positives but might result in m
...
+> - All policies with min accuracy or custom SIT patterns with confidence levels less than equal to 65 will be mapped to low confidence.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-learn-about.md]:
```suggestion
> - All policies with minimum accuracy or custom SIT patterns with confidence levels less than or equal to 65 will be mapped to low confidence. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by Josegto1 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by JohanFreelancer9** [comment]
@arastogi205 - please commit my suggestions.

**Comment by arastogi205** [comment]
@JohanFreelancer9 - done. Thanks for the recommended changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**